### ISSUE TITLE
about predict & Brier score calculation

### ISSUE THREAD

**Issue by Wei-777-Wei** [bips-hb/survnet#1]
Dear author，
        I'm writing to ask two questions: 1) what is "survnet::predict " code predict? hazard probability or survival ranks. 2) could the package predict hazard probability at different time point?  That is, is there a way to calculate Brier score or time-depedent calculation curve for prediction model?

Best regards
Betty

**Comment by tagteam** [comment]
Dear Betty, there are hazard rates and survival probabilities but no hazard probabilties and a prediction cannot be a rank either. at this point survnet is work in progress but you can have a look at the CRAN package riskRegression::Score which calculates the time-dependent Brier score for risk prediction models.
best regards, Thomas

**Comment by Wei-777-Wei** [comment]
Hi Thomas,
thanks for your answer. in your answer, the survnet could calculate survival probabilities at different timepoint to calculate the time-dependent Brier score using   riskRegression::Score. Is that what I am to understand? Furthermore, would you share some papers using survnet ? Thank you very much.
Best regards
Betty

**Comment by tagteam** [comment]
no: I was saying that survnet is work in progress. the only paper where we have used survnet is this one here: https://rss.onlinelibrary.wiley.com/doi/full/10.1111/rssa.12644

**Comment by mnwright** [comment]
The output of `survnet::predict()` are cumulative incidences for each time interval (and cause if competing risks). See also Section 4.3 in the paper mentioned  by Thomas. 

Based on the example code from survnet: 
````R
library(survival)
library(survnet)

# Survival data
y <- veteran[, c(3, 4)]
x <- veteran[, c(-2, -3, -4)]
x <- data.frame(lapply(x, scale))
breaks <- c(1, 50, 100, 200, 500, 1000)

# Fit simple model
fit <- survnet(y = y, x = x, breaks = breaks)
````

You can use `riskRegression::Score()` like this to calculate things like the Brier score: 
````R
library(riskRegression)
pred <- predict(fit, x)
Score(list("myANN" = pred), formula = Surv(time,status) ~ 1, 
      data = y, times = breaks)
````
